thumbor/handlers/__init__.py

```diff
@@ -620,6 +620,8 @@ def _fetch(self, url):
             is_mixed_no_file_storage = is_mixed_storage and isinstance(storage.file_storage, NoStorage)
 
             if not (is_no_storage or is_mixed_no_file_storage):
+                self.context.config = copy.deepcopy(self.context.config)
```

Comment:
@mb-m The only thing that I'd change here is add a comment on to why we are copying a config locally. It would make clear to anyone that reads it later on.